site/docs/pages/guides/telemetry.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Telemetry · OnchainKit
+description: Understanding OnchainKit's anonymous telemetry system and how to configure it.
+---
+
+## Telemetry
+
+OnchainKit is introducing an anonymous telemetry system to help us better understand how our library is used in the wild. Participation in this anonymous program is optional—if you'd prefer not to share any usage data, you can easily opt out.
+
+### Why Are We Collecting Telemetry?
+
+OnchainKit has quickly become a go-to full‑stack component library for integrating essential onchain functionality (like `<wallet />`, `<transaction />`, and `<mintNFT />`) in minutes. Until now, our usage insights have been limited to public npm download counts and API endpoint usage. By collecting telemetry data, we can:
+
+- **Gauge Component Usage**: Understand which components (and their variants) are most popular
+- **Data-Informed Improvements**: Help our data science team generate insights that drive future enhancements and refactoring decisions
```

Comment:
nit: since these are continuing the sentence "we can:" ... all bullets should start  with a verb. 

"make data informed improvements"
(not sure on the last one, but could be) "Add proactive monitoring"